pgmpy/sampling/Sampling.py

```diff
@@ -87,6 +88,7 @@ def forward_sample(self, size=1, return_type='dataframe'):
                 weights = cpd.values
             sampled[node] = sample_discrete(states, weights, size)
 
+        sampled = _map_to_state_name(self.model, sampled)
```

Comment:
I do think we should mention this in the docstring that it returns state names if defined.

Reply:
@ankurankan Okay. But I think users will assume that it will return state name because he would want that. But no harm in writing.

Reply:
@khalibartan Yeah, true. But it's always better to be explicit.